content/ja/mate-desktop/applications/engrampa.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Engrampa - 書庫マネージャ
+weight: -20
+---
+
+<span class="badge-placeholder">[![Build Status](https://travis-ci.org/mate-desktop/engrampa.svg?branch=master)](https://travis-ci.org/github/mate-desktop/mate-desktop)</span>
+<span class="badge-placeholder">[![GitHub release](https://img.shields.io/github/v/release/mate-desktop/engrampa)](https://github.com/mate-desktop/mate-desktop/releases/latest)</span>
+<span class="badge-placeholder">[![GitHub contributors](https://img.shields.io/github/contributors/mate-desktop/engrampa)](https://github.com/mate-desktop/engrampa/graphs/contributors)</span>
+<span class="badge-placeholder">[![License](https://img.shields.io/github/license/mate-desktop/engrampa)](https://github.com/mate-desktop/engrampa/blob/main/LICENSE)</span>
+
+[Source Code](https://github.com/mate-desktop/engrampa) | [Bug Tracker](https://github.com/mate-desktop/engrampa/issues) | [Dependencies](https://github.com/mate-desktop/engrampa/blob/master/.build.yml)
+
+![](https://raw.githubusercontent.com/mate-desktop/engrampa/master/data/icons/scalable/apps/engrampa.svg)
+
+{{< columns >}}
+
+Engrampaは、MATE Desktop Environment の公式なアーカイブマネージャです。tar や zip などのファイル圧縮ユーティリティのグラフィカルなフロントエンドです。これは File Roller のフォークです。
+
+Engrampa では、書庫（アーカイブ）からのファイル作成、変更および展開など最も一般的な操作をサポートします。また、書庫の内容を一覧表示したり、書庫に含まれているファイルを開いたりすることもできます。
+
+    <--->
+
+[![](../images/engrampa-window.png)](../images/engrampa-window.png)
```

Comment:
I moved the images to the static dir. Please replace this with  `![](/img/applications/engrampa-window.png)`

Reply:
Seems that concern is open?

Reply:
@mbkma Maybe we merge that and fix the link after?